scipy/optimize/tests/test_linprog.py

```diff
@@ -1903,14 +1890,14 @@ class TestLinprogRSBland(LinprogRSTests):
 #######################################
 
 
-class TestLinprogHiGHSSimplex(LinprogHiGHSTests):
+class TestLinprogHiGHSSimplexDual(LinprogHiGHSTests):
     method = "highs-simplex"
     options = {}
 
 
-#######################################
-# HiGHS-Simplex Option-Specific Tests #
-#######################################
+###################################
+# HiGHS-IPM Option-Specific Tests #
```

Comment:
? The method is simplex above?

Reply:
I removed the primal simplex tests but left the dual simplex tests.  Is there something missing?

Reply:
Sorry I misread something.